wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -288,6 +288,18 @@ void CommandScheduler::UnregisterSubsystem(
   }
 }
 
+const std::vector<Subsystem*> CommandScheduler::GetAllRegisteredSubsystems() {
+  std::vector<frc2::Subsystem*> subsystems;
+  subsystems.reserve(m_impl->subsystems.size());
+  for (wpi::DenseMap<frc2::Subsystem*, std::unique_ptr<Command>>::iterator it =
+           m_impl->subsystems.begin();
+       it != m_impl->subsystems.end(); ++it) {
+    subsystems.push_back(it->first);
+  }
+  const std::vector<frc2::Subsystem*> ret = subsystems;
```

Comment:
We can just `return subsystems;` without making a copy- `subsystems` is already a copy, and we can convert to a const vector (if we want to) just by returning with the return type specified as const vector.